packages/dds/task-manager/src/test/taskManager.spec.ts

```diff
@@ -149,106 +152,113 @@ describe("TaskManager", () => {
 
             taskManager2.abandon(taskId);
-            // Will reject due to exiting the queue without first acquiring lock
+            // Will reject due to exiting the queue without first acquiring task
             // Promise should be settled already prior to processing messages
             await assert.rejects(volunteerTaskP2);
             containerRuntimeFactory.processAllMessages();
             assert.ok(!taskManager2.queued(taskId), "Should not be queued");
-            assert.ok(!taskManager2.haveTaskLock(taskId), "Should not have lock");
+            assert.ok(!taskManager2.assignedTask(taskId), "Should not be assigned");
         });
 
         it("Can abandon and immediately attempt to reacquire a task", async () => {
             const taskId = "taskId";
-            const lockTaskP = taskManager1.volunteerForTask(taskId);
+            const volunteerTaskP = taskManager1.volunteerForTask(taskId);
             assert.ok(taskManager1.queued(taskId), "Should be queued");
-            assert.ok(!taskManager1.haveTaskLock(taskId), "Should not have lock");
+            assert.ok(!taskManager1.assignedTask(taskId), "Should not be assigned");
             containerRuntimeFactory.processAllMessages();
-            await lockTaskP;
+            const isAssigned = await volunteerTaskP;
+            assert.ok(isAssigned, "Should resolve true");
             assert.ok(taskManager1.queued(taskId), "Should be queued");
-            assert.ok(taskManager1.haveTaskLock(taskId), "Should have lock");
+            assert.ok(taskManager1.assignedTask(taskId), "Should be assigned");
 
             taskManager1.abandon(taskId);
             assert.ok(!taskManager1.queued(taskId), "Should not be queued");
-            assert.ok(!taskManager1.haveTaskLock(taskId), "Should not have lock");
-            const relockTaskP = taskManager1.volunteerForTask(taskId);
+            assert.ok(!taskManager1.assignedTask(taskId), "Should not be assigned");
+            const revolunteerTaskP = taskManager1.volunteerForTask(taskId);
             assert.ok(taskManager1.queued(taskId), "Should be queued");
-            assert.ok(!taskManager1.haveTaskLock(taskId), "Should not have lock");
+            assert.ok(!taskManager1.assignedTask(taskId), "Should not be assigned");
             containerRuntimeFactory.processAllMessages();
-            await relockTaskP;
+            const isAssigned2 = await revolunteerTaskP;
+            assert.ok(isAssigned2, "Should resolve true");
             assert.ok(taskManager1.queued(taskId), "Should be queued");
-            assert.ok(taskManager1.haveTaskLock(taskId), "Should have lock");
+            assert.ok(taskManager1.assignedTask(taskId), "Should be assigned");
         });
 
-        it("Can attempt to lock twice and abandon twice (after ack)", async () => {
+        it("Can attempt to lock task twice and abandon twice (after ack)", async () => {
```

Comment:
Nit here and 2 other instance below
```suggestion
        it("Can attempt to volunteer for task twice and abandon twice (after ack)", async () => {
```